setup.py

```diff
@@ -42,16 +42,16 @@ def get_version(version_file):
 
 
 setup(
-    name="example",
+    name="hash-http-content",
     # Versions should comply with PEP440
-    version=get_version("src/example/_version.py"),
-    description="Example python library",
+    version=get_version("src/hash_http_content/_version.py"),
+    description="HTTP content hasher",
     long_description=readme(),
     long_description_content_type="text/markdown",
     # NCATS "homepage"
     url="https://www.us-cert.gov/resources/ncats",
     # The project's main homepage
-    download_url="https://github.com/cisagov/skeleton-python-library",
+    download_url="https://github.com/cisagov/hash-http-content",
     # Author details
     author="Cyber and Infrastructure Security Agency",
     author_email="ncats@hq.dhs.gov",
```

Comment:
This email address needs to be updated upstream.

Reply:
https://github.com/cisagov/skeleton-python-library/issues/57 Did we reach a consensus on which of each to use? I'm happy to make an upstream PR to resolve this.

Reply:
Please see my comment at https://github.com/cisagov/skeleton-python-library/issues/57#issuecomment-776941344.